### DISCUSSION TITLE
Elasticsearch app is getting crash on ESXi 8.0

### DISCUSSION THREAD

**Discussion by scalaramal** [vmware/govmomi#3237] [Community Help] [answered]
Hello Team, 

I am using Elasticsearch in 7.0 and that is working absolutely fine. Soon I move that same setup to 8.0 it crashes with org.apache.lucene.index.CorruptIndexException. I validated the underline datastore and all other VMs are working fine on the source and destination. 

The same setup is working fine on the 7.0 as I mentioned. I wanted to know if we support 8.0 yet. if yes since when we started supporting it? 

Thank you, 
SA

**Comment by tenthirtyam** [comment, marked as answer]
How's this related to `vmware/govmomi`?
